Abracadabra/abracadabra_web_app/pages/subject/[subject]/search/[search].js

```diff
@@ -0,0 +1,76 @@
+import { Container, Row, Col } from "react-bootstrap";
+import Navbar from "src/components/Navbar.js";
+import QuestionBody from "src/components/QuestionBody.js";
+import DefaultErrorPage from "next/error";
+import SubjectService from "src/services/SubjectService";
+import { useRouter } from "next/router";
+
+function Subject({ subjectName, response }) {
+  const router = useRouter();
+  const { subject, search } = router.query;
+
+  if (response === 404 || response == "failure" || response === 400) {
+    return <DefaultErrorPage statusCode={404} />;
+  }
+
+  return (
+    <>
+      <Navbar subjectTitle={subjectName} />
```

Comment:
Possible to make it dynamic in the future? This basically is using a seperate page for showing the response